### PR TITLE
Network-test: add ip alias while running ported to windows

### PR DESCRIPTION
**Pull request by gut**

### CODE REVIEW

File: daisy_workflows/image_test/network/network-tester.py [hunk 1]
```diff
@@ -39,28 +43,51 @@ def GetHostname(addr, timeout=10):
...
+  info = {
+      'aliasIpRanges': alias_ip_ranges,
+      # a previous fingerprint is needed in order to change the instance
+      'fingerprint': MD.GetInstanceIfaces(instance)[0][u'fingerprint']
+  }
+  MD.Wait(MD.SetInstanceIface(instance, info))
+
+  # The operation is completed but it might not guarantee that the agent has
+  # already applied the rule. For that reason, wait
+  time.sleep(30)
```
Comment (gut) [inline daisy_workflows/image_test/network/network-tester.py]: I didn't like this but I didn't come up with a better method. Anyone?

File: daisy_workflows/image_test/network/network-testee.bat [hunk 2]
```diff
@@ -0,0 +1,40 @@
+@echo off
```
Comment (adjackura) [inline daisy_workflows/image_test/network/network-testee.bat]: Just make this startup script powershell to begin with.

Comment (gut) [reply daisy_workflows/image_test/network/network-testee.bat]: lol, you're right

**Comment by adjackura** [comment]
Does this test pass on Windows? We don't setup ip aliasing in the Windows agent like we do on Linux

**Comment by gut** [comment]
> Does this test pass on Windows? We don't setup ip aliasing in the Windows agent like we do on Linux

Nope, it doesn't route traffic from the aliased IP (I raised https://github.com/GoogleCloudPlatform/compute-image-windows/issues/144 ). If you want me to silently change that behavior for windows, I can do it, but I was hoping to have it fixed so that it also behaves as Linux does.

**Comment by adjackura** [comment]
Updated that issue with some background. I would just skip this test on Windows for now.

**Comment by gut** [comment]
done.

File: daisy_workflows/image_test/network/network-tester.py (hunk 1 above)
Comment (adjackura) [reply daisy_workflows/image_test/network/network-tester.py]: We could watch the serial port, but thats kinda complicated, this sleep is more than enough. If we wanted to clean it up we can later,

**Pull request merged**